### ISSUE TITLE
Potential problem: Modulator behaving weird... 

### ISSUE THREAD

**Issue by Lykdraft** [kmatheussen/radium#1142]
and another one... :)

Looks like the modulator is output is "fluttering"... I first thought this is because I had swing lines going...
but turns out it "flutters" also with swing disabled. (Meaning it goes up and down, but not in a clean way.
It zig-zags quite a bit while doing this)

Now I am not sure if this is only a graphical issue or not. It doesn't look like it though.
(The volume slider representing the modulators modulation is fluttering as well,
as is the modulation indication on the modulator itself, the modulation indicator on the target parameter seems okay though.)

_You should be able to see this very clearly when using high tempo divisor settings. try 16 or 20 for example._

Might want to look into it. Just want to be sure if this is normal or not.

Best,
tL.

**Comment by kmatheussen** [comment]
I'm might see the same thing. It could be because the time information from
the editor is a little bit fluttering. It's complicated.
Does the flutter disappear if you stop playing? Because then the time
information from the editor is simulated and should
not be fluttering. (I don't see any difference though)

**Comment by kmatheussen** [comment]
> So I think we can close this?

The "fluttering" you see, is this only the line stopping up now or then, or
does it actually move in the opposite direction for a short glimps now and
then?
If it just halts now and then, it's probably just the graphics, but if it
even goes backwards, it's likely to be something worse.

**Comment by Lykdraft** [comment]
> does it actually move in the opposite direction for a short glimps now and
> then?

Yes, indeed it does do that.

**Comment by kmatheussen** [comment]
> does it actually move in the opposite direction for a short glimps now and
> then?
>
> Yes, indeed it does do that.
>

I haven't seen that. Can you make a recipe how to reproduce?

**Comment by Lykdraft** [comment]
Aeh, I just add a modulator... :) It immediately does behave like that.

**Comment by Lykdraft** [comment]
But again, the target parameter looks fine. I'll try to take a look at this again later.

**Comment by kmatheussen** [comment]
Do you see it goes backwards now and then if you do this:

1. Right click in the mixer and select "Modulator"
2. In the instrument window, right click the "Manual Input" slider and
select "Assign modulator" and select the modulator itself.
3. Set "Tempo divisor" to 32.
4. The "Manual input" cursor changes direction now and then (when it's not
supposed to).

Are you sure it's not just an optical illusion? When the cursor movement is
not smooth, I guess it could look like the cursor moves slightly backwards
when it stutters.

**Comment by kmatheussen** [comment]
It is strange that it's not smoother though. If you set "block multiplier"
to 0.04, it's very obvious that the movement is not smooth.
Each 1/2 second, approximately, it jumps one step forward instead of
gliding smoothly.

**Comment by kmatheussen** [comment]
Actually, it does move smoothly. But each 1/2 second, it takes a 1/2 second
break, for some reason.

**Comment by Lykdraft** [comment]
Check this video, it's easier to see than tell I think:
(look especially at the Volume & Output Modulation Indicator in the lower right)

https://www.dropbox.com/s/fincwz534vxp82l/Modulator1.mov?dl=0

**Comment by kmatheussen** [comment]
Yes, I see. It's only the audio out that goes backwards though. I guess
that's only got something to do with the audio meter algorithm.

**Comment by Lykdraft** [comment]
Alright. As the target modulation seems fine I think this can be closed?

**Comment by kmatheussen** [comment]
No, keep it open. Something is wrong. and I'm not entirely sure it's just
graphics since both
audio and the "cursor" (need a name for that) both behaves strangely.
